test/spec/PalindromSpec.js

```diff
@@ -9,10 +9,10 @@ describe("Palindrom", function () {
   });
 
   describe("backward compatibility", function () {
-    it("should work if PuppetJs constructor is used", function (done) {
+    it("should work if Puppet constructor is used", function (done) {
       var initSpy = jasmine.createSpy();
 
-      this.palindrom = new PuppetJs({remoteUrl: window.location.href, callback: initSpy});
+      this.palindrom = new Puppet({remoteUrl: window.location.href, callback: initSpy});
```

Comment:
Sure sure, [the constructor is called `Puppet`](https://github.com/Palindrom/Palindrom/blob/2.4.x/src/puppet.js#L521) it was just a mistake in the spec.